README.md

```diff
@@ -23,7 +23,11 @@ Includes:
 
 ## Setup
 
-1. `yarn && yarn start` - To open Storybook
+1. `yarn` - install dependencies
+2. `yarn start` - To open Storybook
+
+### Advanced
+- define the Maps API key and run storybook using this command: `STORYBOOK_MAPS_API_KEY=[you long api key] yarn start` - see: (https://storybook.js.org/docs/configurations/env-vars/) for more info
```

Comment:
Is there no better way to do this?
might get a bit tedious with more env variables

Reply:
no better way, since its a public repo - no api keys should be committed - also the key is not required to view - it's more for dev purposes